README.md

```diff
@@ -1,10 +1,9 @@
-ZF Content Validation
-=====================
+# ZF Content Validation
```

Comment:
We cannot change headings in this file, as doing so affects the documentation parser we use on apigility.org.